Test/AwsCommonRuntimeKitTests/http/HttpClientConnectionManagerTests.swift

```diff
@@ -3,17 +3,17 @@
 import XCTest
 @testable import AwsCommonRuntimeKit
 
-class HttpClientConnectionManagerTests: CrtXCBaseTestCase {
+class HttpClientConnectionManagerTests: XCBaseTestCase {
```

Comment:
Why are these classes still "Http" and not "HTTP"?
it looked like we were doing all-caps acronyms

Reply:
Thanks, I have updated it because HTTP commonly appears as all upper case. 

Swift’s official guide states: “Acronyms and initialisms that commonly appear as all uppercase in American English should be uniformly up- or down-cased according to case conventions. Other acronyms should be treated as ordinary words.” 

Reference: https://www.swift.org/documentation/api-design-guidelines/#general-conventions